src/Components/Authentication/commonElements.tsx

```diff
@@ -1,3 +1,5 @@
+// @ts-ignore
```

Comment:
Ooh so it's legit to ignore or is this meant to be temporary?

Reply:
Yeah it's because of how Typescript type emission work, a fix is being considered (https://github.com/Microsoft/TypeScript/issues/9944) but in the meantime that's the recommended way to silence the compiler.

Here's a longer explanation https://github.com/artsy/reaction/pull/715/files#r193201970